### ISSUE TITLE
vue3-dnd-test-utils, peerDependency "@testing-library/vue": ">= 11" is unknown

### ISSUE THREAD

**Issue by schmulschubiak** [hcg1023/vue3-dnd#50]
Hejhej,
the **vue3-dnd-test-utils** mention the need of the peerDependency ``"@testing-library/vue": ">= 11"``, but the latest version I can find is 7.0.0 (https://github.com/testing-library/vue-testing-library/tags)
Did I'm missing something? Or is this the wrong dependency?
Thx! :)

**Comment by hcg1023** [comment]
Sorry, this is a wrong dependency version and needs to be modified

**Comment by schmulschubiak** [comment]
Ah I see, maybe copy&pasted from the react version https://github.com/react-dnd/react-dnd/blob/main/packages/test-utils/package.json

**Comment by hcg1023** [comment]
Yes, to be honest, I didn't pay too much attention to test-utils, but now it seems that I may need to deal with the vue version here

**Comment by hcg1023** [comment]
Now the dependency issues have been dealt with, but there are still issues that cannot be rendered correctly during testing, waiting for the next step of troubleshooting.
